Domains.md

```diff
@@ -0,0 +1,46 @@
+# Community domains
+
+This document sets out rules for community domains and the responsibilities and expectations of anyone that maintains and manages the domain settings of official domains used by the Bitcoin Design community. Community domains are used to promote and host content that the community has created and committed to the github.com/bitcoindesign repositories. This currently applies to;
+
+- bitcoin.design
+- bitcoindesigners.org
+
+## The community
+The Bitcoin Design community, defined as; 
+
+*“The active members of the Slack workspace at bitcoindesign.slack.com, represented in rough-consensus by the maintainers of the BitcoinDesign Github organization at github.com/bitcoindesign”.*
+
+## Ownership
+A community domain shall be considered owned by the Bitcoin Design community.
+The role of domain maintainer does not imply any ownership of the domain. 
+
+## Registration and payments
+The community will pay for, or solicit donations to pay for the long term registration of the community domain(s). 
+
```

Comment:
perhaps we state the renewal costs for the domains for transparency?

bitcoin.design is $49.88 a year.
bitcoindesigners.org is $12 ("plus tax")

Reply:
Added.
Once bitcoin.design maintainer is updated we should also update the renewal date, if say we do a 10 year up-front renewal.